### PR TITLE
Add HW-27 Solution

### PR DESCRIPTION
**Pull request by BilkoYevgen**

### CODE REVIEW

File: HW-27/hw27.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+// 2. Додати окремий елемент, що має відображати кількість друзів. Ця кількість має ініціалізуватися як рандомне число при завантаженні сторінки, а при натисканні на "Додати в друзі" збільшуватися на 1.
+
+var friendCount = Math.floor(Math.random() * 100) + 1;
+
+var addFriendBtn = document.getElementById("addFriendBtn");
+var friendCountSpan = document.getElementById("friendCount");
+
+friendCountSpan.innerHTML = friendCount + " друзів";
+
```
Comment (klymenok) [inline HW-27/hw27.js:28]: The button should become unclickable (disabled) after the first click.

File: HW-27/hw27.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+jobOfferBtn.innerHTML = "Запропонувати роботу";
+jobOfferBtn.style.background = "purple"
+jobOfferBtn.addEventListener("click", function () {
+    alert("Відкривається форма для заповнення пропозиції роботи.");
+});
+document.body.appendChild(jobOfferBtn);
+
+// 2. Додати окремий елемент, що має відображати кількість друзів. Ця кількість має ініціалізуватися як рандомне число при завантаженні сторінки, а при натисканні на "Додати в друзі" збільшуватися на 1.
+
+var friendCount = Math.floor(Math.random() * 100) + 1;
```
Comment (klymenok) [inline HW-27/hw27.js]: It is better to use `let` instead of `var`

File: HW-27/hw27.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+addFriendBtn.addEventListener("click", function () {
+
+    friendCount++;
+    friendCountSpan.innerHTML = friendCount + " друзів";
+});
+
+//  Зробити неможливим повторне натискання кнопки “Додати в друзі”, змінювати її назву на “Очікується підтвердження” одразу після натискання і робити неактивною.
+
+// var friendCount = Math.floor(Math.random() * 100) + 1;
```
Comment (klymenok) [inline HW-27/hw27.js]: Remove these comments or uncomment for evaluation

File: HW-27/hw27.js (hunk 1 above)
Comment (klymenok) [reply HW-27/hw27.js:28]: And the text should be changed according to the task

**Review by klymenok — CHANGES_REQUESTED** [review]

File: HW-27/hw27.js (hunk 1 above)
Comment (BilkoYevgen) [reply HW-27/hw27.js:28]: Add some changes on js file.

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**